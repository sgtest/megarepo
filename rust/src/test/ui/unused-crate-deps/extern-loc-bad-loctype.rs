// --extern-location with bad location type

// aux-crate:bar=bar.rs
// compile-flags:--extern-location bar=badloc:in-the-test-file

#![warn(unused_crate_dependencies)]

fn main() {}
