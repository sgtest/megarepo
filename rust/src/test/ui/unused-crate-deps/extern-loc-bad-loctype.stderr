error: unknown location type `badloc`: use `raw` or `json`

